src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -153,6 +153,56 @@ TEST_CASE("CheckCopyMovingVanillaNetworkTest", "[FeedForwardNetworkTest]")
   CheckMoveFunction<>(model1, trainData, trainLabels, 1);
 }
 
+/**
+ * Concatenate layer constructor test.
+ */
+TEST_CASE("CheckCopyMovingConcatenateTest", "[FeedForwardNetworkTest]")
+{
+  // Create training input by 5x5 matrix
```

Comment:
Hey @rxng8,
Could you also add a stop at the end sentences. Thanks.

Reply:
Thank you for reminding me! Resolved!